### PR TITLE
Use prop() in place of attr() for disabled and checked.

### LINKED ISSUE #1208: Replace .attr() with .prop() in common/upload-addon.js
http://192.168.99.100:8000/en-us/developers/addon/submit/2

Relates to #1197

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #1208

### CODE REVIEW

**Comment by tofumatt** [comment]
r+

File: static/js/common/upload-addon.js [hunk 1]
```diff
@@ -344,7 +344,7 @@
             updateListedStatus();
 
             $('#id_is_manual_review').bind('change', function() {
-                $('.addon-upload-dependant').attr('disabled', !($(this).is(':checked')));
+                $('.addon-upload-dependant').prop('disabled', !($(this).is(':checked')));
```
Comment (magopian) [inline static/js/common/upload-addon.js]: Do we want `.prop('checked')` here for coherence?

**Comment by magopian** [comment]
r+wc

File: static/js/common/upload-addon.js (hunk 1 above)
Comment (muffinresearch) [inline static/js/common/upload-addon.js]: Yes probably but following the rule we made yesterday about not trying to fix cruft I'm going to err on the side of not making clean-up changes right now.

**Pull request merged**